### PR TITLE
Run Cromwell in CWL test dir

### PR DESCRIPTION
**Pull request by Horneth**
With https://github.com/broadinstitute/cromwell/pull/2988 it should enable a bunch of new CWL conformance tests to run on travis. They will still need the outputs to look right to fully "pass"

### CODE REVIEW

File: src/bin/travis/testCentaurCwlConformance.sh [hunk 1]
```diff
@@ -19,8 +20,12 @@ shutdownCromwell() {
 
 trap "shutdownCromwell" EXIT
 
+# Start the Cromwell server in the directory containing input files so it can access them via their relative path
+CURRENT_DIR=$(pwd)
+cd CWL_TEST_DIR
 java -jar ${CROMWELL_JAR} server &
 CROMWELL_PID=$$
+cd CURRENT_DIR
```
Comment (mcovarr) [inline src/bin/travis/testCentaurCwlConformance.sh]: you might subshell this instead

Comment (Horneth) [reply src/bin/travis/testCentaurCwlConformance.sh]: If I subshell it will `CROMWELL_PID` still be defined outside the subshell ?

Comment (mcovarr) [reply src/bin/travis/testCentaurCwlConformance.sh]: ugh nm. you can do disgusting things with globals but that's totally not worth it.

**Comment by cjllanwarne** [comment]
I was going to say "rebase on `develop` to unredden TES". Unfortunately the CWL conformance tests are also red (and since we ignore the results right now, that means they are failing to run at all, I believe)

**Comment by kshakir** [comment]
Aha, [a week ago](https://github.com/common-workflow-language/common-workflow-language/commit/915d49f8fb912774cbf5cbe6ba910a430ef69941) 100% of tests were moved under the `v1.0` directory. So this "fix" should now work!

**Pull request merged**